openssl-dynamic/src/main/c/ssl.c

```diff
@@ -1442,10 +1368,7 @@ TCN_IMPLEMENT_CALL(void, SSL, setOptions)(TCN_STDARGS, jlong ssl,
 {
     SSL *ssl_ = J2P(ssl, SSL *);
 
-    if (ssl_ == NULL) {
-        tcn_ThrowException(e, "ssl is null");
-        return;
-    }
+    TCN_CHECK_NULL(ssl_, "ssl is null",);
```

Comment:
consider clarifying this as follow: `TCN_CHECK_NULL(ssl_, "ssl is null", /*void*/);`